test/parallel/test-trace-events-api.js

```diff
@@ -3,7 +3,7 @@
 
 const common = require('../common');
 
-if (!common.hasTracing)
+if (!process.binding('config').hasTracing)
```

Comment:
This could likely be modified to simply attempt to `require('trace_events')`, which will throw if tracing is not enabled... 

```js
try {
  require('trace_events');
} catch (err) {
  common.skip('missing trace events');
}
```

Reply:
I don't oppose such a modification, but if given the choice, I'd prefer to keep this a simple migrate-the-existing-thing-into-the-one-test-that-uses-it and have any modifications to the-existing-thing happen in a subsequent PR.